src/System.Net.Http/src/netcore50/System/Net/HttpClientHandler.cs

```diff
@@ -167,99 +168,48 @@ public bool PreAuthenticate
 
         public bool UseDefaultCredentials
         {
-            get { return Credentials == null; }
+            get { return false; }
```

Comment:
I thought that if the app has Enterprise Auth it will use default credentials and credentials haven't been set. Can we somehow implement capability detection and return the correct state?